hooks/blueprint

```diff
@@ -29,6 +29,7 @@ done
 merge+=( \
   overlay/base.yml \
   overlay/uaa-clients.yml \
+  overlay/operations/rename-network-and-deployment.yml \
```

Comment:
this is an ops file, not an overlay, so it should be in `ops/` or `operations/` not `overlay/operations/`